libraries/Microsoft.Bot.Builder.Dialogs/Microsoft.Bot.Builder.Dialogs.csproj

```diff
@@ -46,6 +46,10 @@
 		<DocumentationFile>bin\Documentation\netstandard2.0\Microsoft.Bot.Builder.Dialogs.xml</DocumentationFile>
 	</PropertyGroup>
 
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
I don't think we want to build docs on normal Debug builds... that goes against another change we had already made. There's already a `Documentation` configuration property group above that adds `<DocumentationFile>` when that configuration is selected.